cmd/oras/tag/tag.go

```diff
@@ -42,19 +49,24 @@ func TagCmd() *cobra.Command {
 Example - Tag the manifest 'v1.0.1' in 'locahost:5000/hello' to 'v1.0.2':
   oras tag localhost:5000/hello:v1.0.1 v1.0.2
 
-Example - Tag the manifest with digest sha256:9463e0d192846bc994279417b50114606712d516aab45f4d8b31cbc6e46aad71 to 'v1.0.3'
-  oras tag localhost:5000/hello@sha256:9463e0d192846bc994279417b50114606712d516aab45f4d8b31cbc6e46aad71 v1.0.3
+Example - Tag the manifest with digest sha256:9463e0d192846bc994279417b50114606712d516aab45f4d8b31cbc6e46aad71 to 'v1.0.2'
+  oras tag localhost:5000/hello@sha256:9463e0d192846bc994279417b50114606712d516aab45f4d8b31cbc6e46aad71 v1.0.2
```

Comment:
Should we add an example for tagging multiple tags?

Reply:
added an example for tagging multiple tags with default concurrency number.